frontend/src/components/Banner.js

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2021 Christoph Ehm <christoph.ehmendoerfer@campus.tu-berlin.de>
+
+import React, { useEffect, useState } from "react";
+import HeaderTypography from "./HeaderTypography";
+import ParagraphTypography from "./ParagraphTypography";
+import { Link } from "react-router-dom";
+//import appwriteApi from "../api/appwriteApi";
+import RoundedEdgesButton from "./RoundedEdgesButton";
+import { Grid } from "@mui/material"
+
+import { textColor, signupButtonColor } from "../assets/jss/colorPalette";
+import bannerImage from "../assets/img/banner.png"
+
+const backgroundImageStyle = {
+	backgroundImage: `url(${bannerImage})`,
+	backgroundPosition: "center",
+};
+
+/**
+ * Displays a welcoming background image with a button.
+ * @param user user object of the currently logged in account
+ * @returns {JSX.Element}
+ */
+export default function Banner({ children, title, user }) {
+	const bannerTitle = <BannerTitle>{ title }</BannerTitle>;
+	const bannerSubTitle = <BannerSubTitle>{ children }</BannerSubTitle>;
+	const bannerButton = <BannerButton isLoggedIn={(user != null)} style={{ fontSize: 17, }}/>;
+	const bannerContainerStyle = {
+		height: 305,
+		width: 1100,
+		left: 164,
+		top: 96,
+		borderRadius: 15,
+		padding: 40,
+		textAlign: "center",
+	};
```

Comment:
Do not hardcode any width. This destoys responsive designs. Instead use something like `100%` or `100vw` or other ratios.

Reply:
Also, personaly I would just put these style elements into the component directly. But thats mostly up to you.

Reply:
Done (?)